app/views/katello/api/v2/content_view_components/show_all.json.rabl

```diff
@@ -0,0 +1,12 @@
+object false
+
+extends "katello/api/v2/common/metadata"
+
+child @collection[:results] => :results do
+  extends 'katello/api/v2/content_views/base'
+  node(:added_to_content_view) { |cv| cv.component_of?(@view) }
+  node(:cv_versions) do |cv|
+    @version = cv.component_content_view_version(@view)
+    partial('katello/api/v2/content_view_components/show', :object => @version)
+  end
+end
```

Comment:
i'm trying to understand the reasoning behind what's included.  The previous example of this (repositories/show_all) extended the repostiories/base partial and 'enhanced' it by adding 'added_to_content_view'.  Can we do something similar here?

Reply:
Updated as per our discussion yesterday to use a presenter and the existing index template.